src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentQueue.cs

```diff
@@ -900,7 +901,8 @@ internal int High
                 {
                     //if _high > SEGMENT_SIZE, it means it's out of range, we should return
                     //SEGMENT_SIZE-1 as the logical position
-                    return Math.Min(_high, SEGMENT_SIZE - 1);
+                    var high = _high; // jit won't inline Math.Min if passed a volatile
```

Comment:
It just came up as a weird thing not to be inlining when I was doing inlining traces for something else. It is about 8 inlines so it will have some effect; but say you say likely to be lost in bigger noise.

However, it seems to be a fairly mundane change in a hot path. So low risk, mild benefit :smile: